website/docs/shadertype.md

```diff
@@ -1,24 +1,24 @@
-shadertype - This page was auto-generated. Feel free to help us improve the documentation by creating a pull request.
+Specifies the type of shader.
 
 ```lua
-shadertype (value)
+shadertype "value"
```

Comment:
Let's use `shadertype ("value")` instead. It's more consistent with other docs.

Reply:
+1